### DISCUSSION TITLE
Playback Stops After Last Playlist Item Finishes - Cannot Play Anything Again Without Restart

### DISCUSSION THREAD

**Discussion by BzowK** [MiczFlor/RPi-Jukebox-RFID#1060] [Q&A]
### The Issue

This is a newly built Phoniebox built on Buster with one-line command.  After boot, everything works great except audio stops during playback randomly after a few minutes.  At the same time, the volume circle display in player web interface is empty instead of showing a percentage.  

I did bit of troubleshooting to find that when running `systemctl status mpd |grep 'Active: '|sed 's/Active: //g'`, the result was inactive (dead).  I couldn't find any relevant phonebox (specific) logs and when examining mpd.log files, one was empty and the other full of a few random charachters.

### Further information that might help
The build is a RasbPi 3 with USB audio connector & USB-powered speakers.  Also has RFID (MPC522) reader, two pushbuttons, & rotary encoder.  Powered by 5.2V 3A.  Wondering if need to directly power speakers instead of via USB if causing issue.

### Logs
[Here's a link to the latest syslog as just the last 500 lines didn't seem to have enough info](https://kellermannnetwork.com/owncloud/s/rdv8GR18xlLOksZ)

[Here's the mopidy output (if different)](https://kellermannnetwork.com/owncloud/s/sitYctkriCd18l0)

Thanks!

**Comment by BzowK** [comment]
UPDATE

So after more testing, I found the issue is a bit different than first thought.  The issue of playback stopping / MDP stopping / volume not displayed with Phoniebox is caused not randomly, but when the last track of a playlist finishes.  Attempting to start another playlist does not play anything nor even show it attempting to play in the web console.

Please advise - Thanks

**Comment by s-martin** [comment]
I assume you use Spotify edition?

**Comment by BzowK** [comment]
Thanks for the reply...

Whatever the one-line installer for Buster installs.  I reran it today and set back up from scratch (after performing an sudo apt-get update / sudo apt-get upgrade) so should be on the latest version, right?

Thanks

**Comment by s-martin** [comment]
The installer asks you, if you want to install the classic or the Spotify edition.

Please execute `cat /home/pi/RPi-Jukebox-RFID/settings/edition`

**Comment by Freacly** [comment]
Hello,
i've got exactly the same setup and issue, Could someone help?

It stops playback after 30 or 40 minuts. Then  audio stops & the volume circle display in player web interface is empty instead of showing a percentage.

Its not responding to anything at this moment.
When rebooting, its back working for about 30-40 minuts and we are back at the start.

Does someone found a solution? 

Phoniebox version:
Betriebssystem:Raspbian
Beschreibung:Raspbian GNU/Linux 10 (buster)
Release:10
Codename:buster
 Phoniebox Setup
Version2.0 rc7
EditionClassic edition (barebones)
MPD Server Status
inactive (dead)

**Comment by Freacly** [comment]
> Thanks for the reply...
> 
> Whatever the one-line installer for Buster installs. I reran it today and set back up from scratch (after performing an sudo apt-get update / sudo apt-get upgrade) so should be on the latest version, right?
> 
> Thanks

Did you found a solution?

> The installer asks you, if you want to install the classic or the Spotify edition.
> 
> Please execute `cat /home/pi/RPi-Jukebox-RFID/settings/edition`

Do you got any idea to fix it?

**Comment by s-martin** [comment]
What's the output of `tail -n 500 /var/log/syslog`?
